pkg/apply/basic_printer.go

```diff
@@ -23,23 +23,55 @@ type BasicPrinter struct {
 	IOStreams genericclioptions.IOStreams
 }
 
+type applyStats struct {
+	serversideApplied int
+	created           int
+	unchanged         int
+	configured        int
+}
+
+func (a *applyStats) inc(op event.ApplyEventOperation) {
+	switch op {
+	case event.ServersideApplied:
+		a.serversideApplied++
```

Comment:
I wonder --- how do you get feedback in SSA case ? Does the server response indicates that ?

Reply:
The client knows whether it did a serverside or clientside apply. We use the code from upstream kubectl to handle this. AFAIK, we don't know whether a resource was create, configured or unchanged in the serverside apply case.